lib/elixir/lib/dynamic_supervisor.ex

```diff
@@ -22,17 +22,17 @@ defmodule DynamicSupervisor do
 
   The options given in the child specification are documented in `start_link/1`.
 
-  Once the dynamic supervisor is running, we can start children
-  with `start_child/2`, which receives a child specification:
+  Once the dynamic supervisor is running, we can use it to start children
+  on demand. Given the `Stack` example from the `GenServer` documentation,
+  we can use `start_child/2` with a child specification to start a `Stack`:
```

Comment:
I am not sure if it can be confusing to not have the whole context of the Stack implementation here. Perhaps we both refer to it and include its implementation?